src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,33 @@
 package mate.controller;
 
+import java.io.IOException;
+import java.time.LocalDateTime;
+import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
 public class IndexController extends HttpServlet {
     private MyCoolResource myResource;
-    //TODO: implement
+
+    @Override
+    public void init() throws ServletException {
+        super.init();
+        myResource = MyCoolResource.openResource();
+    }
+
```

Comment:
Hi, MAN! YOU #**^*&%!!! It is considered best practice to mark overridden methods with an annotation! You understand this little ^&*%^& !!